app/services/inherited_proofing/va/user_attributes/user_attributes_request.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../../request'
+
+module InheritedProofing
+  module Va
+    module UserAttributes
+      class UserAttributesRequest < InheritedProofing::Request
```

Comment:
We only expect to make the one API request to VAIP for the whole flow, right? I would not bother with making a base class, just creating the one class.

I'd also consider collapsing everything into one service class:

```
service = VaInheritedProofingService.new
attributes = service.user_attributes(token: 'token')
```

Reply:
The way I see it, we don't have a concrete timeline (that I know of) for other IP services, and we don't know how consistent the various services will be with each other, I would stick to just the single class, and we can extract and share common things later

Reply:
@zachmargolis 

"we don't know how consistent the various services will be"

Was the very reason I made the service base class simple and separated out the request and response objects, but I do see your point and I agree it's a good one. Lemme pick @malissal 's brain on this as well, if you don't mind.
 
Ty for your feedback Zach

Reply:
I think I agree with @gangelo here. I think it will be clearer as this feature progresses if we break it out. Future devs can see that VA has specific rules for request and response and it will be easier to see "why" it's done that way. I know we all prefer KISS, but I think in this situation, the added complexity now will improve the onboarding experience for devs working on this feature in the future.

Reply:
I don't feel strongly and am content either way, but I'm sympathetic to Zach's points 🙂

Is there concern that it would be difficult to move to a more abstract set of services if/when we go from 1 to N?  If it's easy enough to defer going down that path and implement when we get to that point, I'd err on the side of simplicity.

Reply:
@mitchellhenke,

It shouldn't be that difficult ultimately, I just like putting the hard work in, upfront; but Zach also makes a good point. I think simple is okay to start. Thank you both for all your feedback and help to think this through!

CC @holytoastr, @zachmargolis,

Reply:
@mitchellhenke,

It shouldn't be that difficult ultimately, I just like putting the hard work in, upfront; but Zach also makes a good point. I think simple is okay to start. Thank you both for all your feedback and help to think this through!

CC @holytoastr, @zachmargolis,

Reply:
@mitchellhenke,

It shouldn't be that difficult ultimately, I just like putting the hard work in, upfront; but Zach also makes a good point. I think simple is okay to start. Thank you both for all your feedback and help to think this through!

CC @holytoastr, @zachmargolis,

Reply:
@mitchellhenke,

It shouldn't be that difficult ultimately, I just like putting the hard work in, upfront; but Zach also makes a good point. I think simple is okay to start. Thank you both for all your feedback and help to think this through!

CC @holytoastr, @zachmargolis,